packages/next/src/generators/next-auth/generator.ts

```diff
@@ -52,6 +68,19 @@ export default async function nextAuthGenerator(
             envVar: options.redisEnvVar,
             name: options.redisAdapterName,
         });
+
+        // Update project.json
+        updateProjectJsonHelmUpgradeTarget(project, tree);
+
+        // Update helm yamls
+        updateDeploymentYaml(project, tree);
```

Comment:
project configuration is updated in `updateProjectJsonHelmUpgradeTarget(project, tree);` and then the pre-updated project configuration is passed into these functions. Pass in the new project configuration `updateDeploymentYaml(readProjectConfiguration(...), tree);`.

Reply:
Also on line 91, as you are updating helm yaml files, this will trigger the attempted format on helm yaml files that are unsupported by prettier. Use the function within @ensono-stacks/core for formatting files and exclude the helm yaml target files from prettier.
```
import {
    formatFiles,
} from '@ensono-stacks/core';    

await formatFiles(tree, [
        globPatternForHelmFiles,
]);
```